src/Std/Time/Zoned/Database/Basic.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Zoned.ZoneRules
+import Std.Time.Zoned.Database.TzIf
+
+namespace Std
+namespace Time
+
+set_option linter.all true
+
+/--
+A timezone database from which we can read the `ZoneRules` of some area by it's id.
+-/
+class Database (α : Type) where
+
+  /--
+  Loads a `ZoneRules` by its id.
+  -/
+  load : α → String → IO TimeZone.ZoneRules
+
+  /--
+  Loads a `ZoneRules` that is set by the local computer.
+  -/
+  localRules : α → IO TimeZone.ZoneRules
```

Comment:
```lean
def main (_ : List String) : IO Unit := do
  let db := Database.TZdb.default
  let rules ← Database.localRules db
  println! "{repr rules}"
```
Here I get an empty array of leap seconds -- is that correct? Same for `Database.load db "Europe/Berlin"`.

Reply:
It's usually empty for some reason :S I tried with https://crates.io/crates/tzif/0.2.3 and it's empty too. But I think that leap seconds are in another file that I just forgot to parse, I'm going to add it.